### ISSUE TITLE
JAWS stops reading output if there is a pause

### ISSUE THREAD

**Issue by JasonMolisani** [microsoft/terminal#15382]
### Windows Terminal version

1.16.10261.0

### Windows build number

10.0.19045.2846

### Other Software

JAWS Version 2022.2211.7 ILM

### Steps to reproduce

I created a very slimmed down project using System.CommandLine [here](https://github.com/JasonMolisani/Problem-with-Windows-Terminal-and-JAWS)

1. Clone the repo and build the project.
2. With Narrator on, run the executable
3. With JAWS on (and Narrator off), run the executable

<img width="606" alt="image" src="https://github.com/microsoft/terminal/assets/60273820/e12b8a93-4675-4dd3-a62a-97c7b9878184">

### Expected Behavior

Regardless of which screen reader is running, I would expect to hear all 16 colors that are printed to the screen. (Yes, it is 16 colors. but the black is being printed with black text, so you can't see it. Screen readers should still read it.)

### Actual Behavior

JAWS does not read the colors that are printed to the screen at all. If I ran this directly in Windows Powershell, not Powershell inside Windows Terminal, JAWS will read the output.

As I was writing this out I had a hunch, so I tried removing the Thread.Sleep() line. Once I did that JAWS started properly reading the output. That may be a place to start looking.

**Comment by zadjii-msft** [comment]
Huh. 2022.2211.7 should have Terminal support (which IIRC was added in 2022.2112.24).

**Comment by zadjii-msft** [comment]
Hey quick question - does this repro in 1.19 [Preview](https://aka.ms/terminal-preview)/? We think this should have been fixed in #14723...

Does updating to a newer version of JAWS help at all here either?
